packages/nouns-webapp/src/hooks/useTreasuryBalance.ts

```diff
@@ -12,7 +12,7 @@ import { BigNumber, ethers } from 'ethers';
  */
 export const useTreasuryBalance = () => {
   const ethBalance = useEtherBalance(config.addresses.nounsDaoExecutor);
-  const lidoBalanceAsETH = useLidoBalance();
+  const lidoBalanceAsETH = useLidoBalance(config.addresses.nounsDaoExecutor);
```

Comment:
NTH: add rocket ETH balance as well

Reply:
can you say more about this?

Reply:
closing this out per our chat in discord